services/database-analyzer.js

```diff
@@ -79,9 +79,14 @@ function DatabaseAnalyzer(databaseConnection, config) {
             const reference = {
               ref: foreignKey.foreign_table_name,
               foreignKey: foreignKey.column_name,
+              foreignKeyname: _.camelCase(foreignKey.column_name),
               as: formatAliasName(foreignKey.column_name),
             };
 
+            if (reference.foreignKeyName === reference.as) {
+              reference.foreignKeyName = `${reference.foreignKeyName}Key`;
```

Comment:
I did a quick test about the `Key` suffix and it reinforces the fact that I am not confident with this change.
99% sure it will break some generated queries in the case of "conventional" column name.
<img width="2086" alt="Screenshot 2019-09-25 at 17 36 07" src="https://user-images.githubusercontent.com/664261/65616632-a9abd580-dfbb-11e9-955e-98b379e0cd31.png">